tools/StaticAnalysis/Exceptions/Az.IotHub/BreakingChangeIssues.csv

```diff
@@ -1 +1,3 @@
 "AssemblyFileName","ClassName","Target","Severity","ProblemId","Description","Remediation"
+"Microsoft.Azure.PowerShell.Cmdlets.IotHub.dll","Microsoft.Azure.Commands.Management.IotHub.GetAzureRmIotHub","Get-AzIotHub","0","3010","The property 'OperationsMonitoringProperties' of type 'Microsoft.Azure.Commands.Management.IotHub.Models.PSIotHubProperties' has been removed.","Add the property 'OperationsMonitoringProperties' back to type 'Microsoft.Azure.Commands.Management.IotHub.Models.PSIotHubProperties'."
```

Comment:
@anusapan as mentioned in the review and by @isra-fel, removing these properties are breaking change and aren't allowed. Please bring back the removed properties.

Reply:
Changes reverted